src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/BreakpointTimelinePoint.tsx

```diff
@@ -27,20 +26,6 @@ function hasPrimaryHighlight({
   return hoveredItem?.point === point?.point;
 }
 
-function hasSecondaryHighlighted({
-  hoveredItem,
-  breakpoint,
-}: {
-  hoveredItem: HoveredItem | null;
-  breakpoint: Breakpoint;
-}) {
-  if (!breakpoint.id || !hoveredItem?.location) {
-    return false;
-  }
-
-  return breakpoint.id == getLocationKey(hoveredItem.location);
-}
```

Comment:
We had a few components and helper functions calculating this "secondary highlight" thing just to assign a CSS class that had no style.